### PR TITLE
avatars and badges

### PR DESCRIPTION
**Pull request by Hnh013**

### CODE REVIEW

File: styles/components/badges.css [hunk 1]
```diff
@@ -0,0 +1,227 @@
...
+
+/* Ring Badge Color Classes */
+
+.ring-eco {    border-color: var(--eco);    }
+.ring-bee {    border-color: var(--bee);    }
+.ring-sun {    border-color: var(--sun);    }
+.ring-fire {    border-color: var(--fire);    }
+.ring-calm {    border-color: var(--calm);    }
+.ring-cool {    border-color: var(--cool);    }
+.ring-over {    border-color: var(--over);    }
```
Comment (SJTGSHIVAM) [inline styles/components/badges.css:199]: Nice usage of classes for color theme I would say.

**Review by SJTGSHIVAM — APPROVED** [review]
Overall it looks clean and coherent to the previous work as well. We can merge this!

**Comment by mdarif6** [comment]
its fine, looking clean

File: docs/avatars.html [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+<!-------------------------------------- CSS Styling Files ------------------------------------------------>
+<!------------------------ Components Styling ---------------------->
+    <link rel="stylesheet" href="../styles/components/avatars.css">
+<!------------------------------------------------------------------>
+<!--------------------------------------------------------------------------------------------------------->
```
Comment (SomeEarth86) [inline docs/avatars.html:11]: comments are spacious and descriptive. 👏

**Review by SomeEarth86 — APPROVED** [review]
looks good

**Pull request merged**